### ISSUE TITLE
TypeError: process.stdout.clearLine is not a function

### ISSUE THREAD

**Issue by neiljso** [atomicals/atomicals-js#90]
When running yarn cli mint-dft electron --satsbyte 50, it's giving below error with the current build. Am I missing something? Thanks!

Starting mining now...

{
  "success": false,
  "message": "TypeError: process.stdout.clearLine is not a function",
  "error": {}
}
Done in 8.70s.

**Comment by neiljso** [comment]
nope, but works fine now. I had problem using git bash but switched to regular cli and everything seems good.

**Comment by Allin-Moondao** [comment]
Just check this PR: https://github.com/atomicals/atomicals-js/pull/136
